geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/SimpleRasterIngest.scala

```diff
@@ -112,7 +111,12 @@ class SimpleRasterIngest(config: Map[String, Option[String]], cs: AccumuloCovera
     val ingestTime = config(IngestRasterParams.TIME).map(df.parseDateTime(_)).getOrElse(new DateTime(DateTimeZone.UTC))
     val metadata = DecodedIndex(Raster.getRasterId(rasterName), bbox.geom, Some(ingestTime.getMillis))
 
-    val res = 1.0  //TODO: get the resolution from the reader or from the gridcoverage
+    val width = rasterGrid.getGridGeometry.getGridRange2D.getWidth
```

Comment:
I'm thinking that this and the duplicate code in the plugin can be factored out into a common function...